src/PrestaShopBundle/Form/Admin/Sell/Product/Pricing/SpecificPriceType.php

```diff
@@ -182,6 +183,9 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
                 'label_tag_name' => 'h4',
                 'required' => false,
                 'has_unlimited_checkbox' => true,
+                'date_format' => 'YYYY-MM-DD HH:mm:ss',
+                'placeholder' => $this->trans('YYYY-MM-DD HH:mm:ss', 'Admin.Global'),
+                'default_end_value' => (new DateTime())->modify('+1 month')->format('Y-m-d H:i:s'),
```

Comment:
We're not supposed to force a default en d value, that was on purpose because some specific prices have no dates specified So I think this particular line changes the behaviour in a bad way

Unless it can still be ignored thanks to the `has_unlimited_checkbox` option?

Reply:
It's not forced, we are just passing our own custom. It was "forced" before in the `DateRangeType.php` itself, see removed line `'data-default-value' => $now->format('Y-m-d'),`.

And yes, if you check "Unlimited", the field is ignored. It works properly as it should. :-)